logspout/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# uses ONBUILD instructions described here:
+# https://github.com/gliderlabs/logspout/tree/master/custom
+
+FROM gliderlabs/logspout:master
```

Comment:
I was following the instructions here to include the logspout to logstash adapter:

https://github.com/gliderlabs/logspout/tree/c2bafe3ea23a2372cf4d72bfc06785d819ad35d7/custom